templates/sag-db-oracle/README.md

```diff
@@ -31,23 +30,21 @@ sagcc exec templates composite apply sag-db-oracle
 
 ## Using for local development and testing on Docker platforms
 
-To use this template for local development launch Command Central container first.
-From the root of this project run:
+To use this template for local development you must first launch the Command Central container. From the root folder of this project run:
 
 ```bash
 docker-compose up -d cc
 ```
 
-Run this command to launch [Oracle Database Express Edition 11g Release 2](https://hub.docker.com/r/wnameless/oracle-xe-11g/)
-container and apply the template to create the user, database and IS, MWS, and BPM schemas for 10.3 release:
+To launch the [Oracle Database Express Edition 11g Release 2](https://hub.docker.com/r/wnameless/oracle-xe-11g/) container and apply the `sag-db-oracle` template to create a user, database, and IS, MWS, and BPM database schemas for the 10.3 release:
```

Comment:
To launch the [Oracle Database Express Edition 11g Release 2](https://hub.docker.com/r/wnameless/oracle-xe-11g/)
container and apply the `sag-db-oracle` template to create a user, storage,
and Integration Server, My webMethods Server and BPM database components for the 10.3 release: